cycle-scripts-es-browserify/scripts/init.js

```diff
@@ -4,14 +4,19 @@ var fs = require('fs-extra')
 var path = require('path')
 var spawn = require('cross-spawn')
 var chalk = require('chalk')
+var ejs = require('ejs')
 
-function dependencies (streamLib) {
-  var basicDependencies = [
-    '@cycle/dom',
-    'xstream'
-  ]
+function dependencies (libs) {
+  var basicDependencies = []
+  var core = {
+    cycle: ['@cycle/dom', 'xstream'],
+    motorcycle: ['@motorcycle/core', '@motorcycle/dom']
+  }
+  var extras = ['immutable', '@cycle/isolate']
```

Comment:
💯 on including `@cycle/isolate` .
Agree that `immutable` should still a user choice and a default of a core-flavor